docs/CustomResources.md

```diff
@@ -236,7 +236,7 @@ the `IndexerCluster` resource provides the following `Spec` configuration parame
 
 ## Example of Guaranteed and Burstable QoS
 
-As given in the [Kubernetes Quality of Service section](readme#using-kubernetes-quality-of-service-classes), you can set up your kubernetes cluster with different Quality of Services by using your Pods’ CPU and Memory resources values. Following are examples of Guaranteed and Burstable QoS:
+As given in the [Kubernetes Quality of Service section](README.md#using-kubernetes-quality-of-service-classes), you can set up your kubernetes cluster with different Quality of Services by using your Pods’ CPU and Memory resources values. Following are examples of Guaranteed and Burstable QoS:
```

Comment:
You can change the CPU and memory resources, and assign different quality of services settings to your pods using the [Kubernetes Quality of Service classes](README.md#using-kubernetes-quality-of-service-classes). Here are some examples of...

Reply:
Done with minor change to use word "classes" instead of "settings" as QoS are coined as classes in K8S.